index.js

```diff
@@ -0,0 +1,54 @@
+let tempBooks = [];
+
+const addBook = () => {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  const book = { id: new Date().getTime(), title, author };
+  if (localStorage.getItem('booksData') !== null) {
+    tempBooks = JSON.parse(localStorage.getItem('booksData'));
+    tempBooks.push(book);
+    const convertedBooks = JSON.stringify(tempBooks);
+    localStorage.setItem('booksData', convertedBooks);
+    window.location.reload();
+  } else {
+    tempBooks.push(book);
+    const convertedBooks = JSON.stringify(tempBooks);
+    localStorage.setItem('booksData', convertedBooks);
+    window.location.reload();
+  }
+};
+
+document.getElementById('addbook').addEventListener('click', addBook);
+
+// eslint-disable-next-line no-unused-vars
+function removeBook(id) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

Reply:
Thank you for your feedback @rlorerh.
This is noted.